packages/xsnap/test/test-xs-perf.js

```diff
@@ -56,7 +63,23 @@ test('meter details', async t => {
     },
     'auxiliary (non-consensus) meters are available',
   );
-  t.is(meterType, 'xs-meter-7');
+  // @ts-ignore extra meters not declared on RunResult (yet?)
```

Comment:
Might be worth tracking this.

Reply:
I'm mentally tracking it as #3139 . I guess since I have to re-spin ci anyway, I'll note it in the comment.

Reply:
fixed in 5fd6c6f